.white-list.yml

```diff
@@ -0,0 +1 @@
+# File for whitelisting frameworks downloaded using carthage_cache: https://github.com/guidomb/carthage_cache
```

Comment:
It is useful. In case we use a framework which name doesn't match its repository name, it should be added to this file, otherwise carthage_cache will remove it.

Take a look at the documentation: https://github.com/guidomb/carthage_cache